### PR TITLE
Fixed Azure Postgresql monitoring setup

### PR DESCRIPTION
**Pull request by laudibert**
### What does this PR do?
Fixes documentation on Azure Postgresql monitoring setup:
- SSL option needs to be set to true
- Username should be in format "<username>@<instance_endpoint>"

### Motivation
I setup this Azure Postgresql monitoring following the documentation and encountered issues. The changes I propose were the I fixes that made it work.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by cswatt** [comment]
Thanks! Added to editorial review queue.

File: content/en/database_monitoring/setup_postgres/azure.md [hunk 1]
```diff
@@ -284,7 +286,7 @@ FROM datadog/agent:7.36.1
 
 LABEL "com.datadoghq.ad.check_names"='["postgres"]'
 LABEL "com.datadoghq.ad.init_configs"='[{}]'
-LABEL "com.datadoghq.ad.instances"='[{"dbm": true, "host": "<AZURE_INSTANCE_ENDPOINT>", "port": 3306,"username": "datadog","password": "<UNIQUEPASSWORD>", "azure": {"deployment_type": "<DEPLOYMENT_TYPE>", "name": "<YOUR_INSTANCE_NAME>"}}]'
+LABEL "com.datadoghq.ad.instances"='[{"dbm": true, "host": "<AZURE_INSTANCE_ENDPOINT>", "port": 3306,"username": "datadog@<AZURE_INSTANCE_ENDPOINT>","password": "<UNIQUEPASSWORD>", "ssl": true, "azure": {"deployment_type": "<DEPLOYMENT_TYPE>", "name": "<YOUR_INSTANCE_NAME>"}}]'
```
Comment (jmeunier28) [inline content/en/database_monitoring/setup_postgres/azure.md:289]: why set `ssl: true`?

Comment (jmeunier28) [reply content/en/database_monitoring/setup_postgres/azure.md:289]: ah okay, it is required by default
> By default, the PostgreSQL database service is configured to require TLS connection. You can choose to disable requiring TLS if your client application does not support TLS connectivity.

... I guess we can set this in our docs & people can turn it off if they decide to disable SSL on the db side

**Review by jmeunier28 — APPROVED** [review]
LGTM

**Review by jtappa — APPROVED** [review]

**Pull request merged**